### PR TITLE
#496: Updated text re file:// URL

### PR DESCRIPTION
**Pull request by samdutton**
See #496.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
 
       <p>Most of the samples use <a href="//github.com/webrtc/adapter">adapter.js</a>, a shim to insulate apps from spec changes and prefix differences. (In fact, the standards and protocols used for WebRTC implementations are highly stable, and there are only a few prefixed names. For full interop information, see <a href="//www.webrtc.org/web-apis/interop">webrtc.org/web-apis/interop</a>.)</p>
 
-      <p>Please note that all samples that use <code>getUserMedia()</code> must be run from a server. Calling <code>getUserMedia()</code> from a file:// URL will result in a PERMISSION_DENIED NavigatorUserMediaError.</p>
+      <p>Please note that all samples that use <code>getUserMedia()</code> must be run from a server. Calling <code>getUserMedia()</code> from a file:// URL won't work.</p>
```
Comment (juberti) [inline index.html]: is an exception thrown, or does it fail silently?

Comment (samdutton) [inline index.html]: Fails silently in Chrome and Opera.

Works in Firefox – don't think it used to. I've updated the text in #496.

**Comment by samdutton** [comment]
Added support for grunt-githooks, which forces grunt to run our validity checks before a git commit is executed (see my email to webrtc-eng).

@phoglund @juberti PTAL.

**Comment by samdutton** [comment]
@phoglund @juberti ping. 

We should probably add grunt-githooks to the other repos as well.

**Comment by KaptenJansson** [comment]
LGTM

**Pull request merged**